doc/classes/RenderingServer.xml

```diff
@@ -436,6 +436,30 @@
 				[b]Note:[/b] The equivalent node is [CanvasItem].
 			</description>
 		</method>
+		<method name="canvas_item_get_instance_shader_parameter" qualifiers="const">
+			<return type="Variant" />
+			<param index="0" name="instance" type="RID" />
+			<param index="1" name="parameter" type="StringName" />
+			<description>
+				Returns the value of the per-instance shader uniform from the specified canvas item instance. Equivalent to [method CanvasItem.get_instance_shader_parameter].
+				[b]Note:[/b] Per-instance shader parameter names are case-sensitive.
```

Comment:
Literally every property is case-sensitive. This note is redundant IMO (I know it's copied).